website/src/hooks/extractBlogPath.ts

```diff
@@ -12,4 +12,10 @@ const useAuthorName = () => {
   return location.pathname.split("/blog/author/").pop();
 };
 
-export { useBlogName, useAuthorName };
\ No newline at end of file
+// custom hook for getting blog tag from the URL
+const useTag = () => {
+  const location = useLocation();
+  return location.pathname.split("/blog/tag/").pop();
```

Comment:
What exactly is this doing ? Do we need this kind of implementation for this.

Reply:
We need this to get the tag name from the URL.  In this `extractBlogPath` hooks, we have similar functions used in blogs to get author name and blog name from URL. So added it here for grouping of functions